jenkins-master/jobs/scripts/workspace/config.py

```diff
@@ -18,11 +20,13 @@
                 'tier': 3,
             },
             'logs': {
```

Comment:
We could have a `raw_log` property for the `buildbot_text` to parse log, and `artifacts` for the rest?